mappings/net/minecraft/entity/unsorted/Mob.mapping

```diff
@@ -0,0 +1 @@
+CLASS none/zi net/minecraft/entity/unsorted/Mob
```

Comment:
This one is similar to the one above. It's the interface used by *all* living entities which are not peaceful. Will add the I prefix.